scripts/docker/buildpack-deps/Dockerfile.emscripten

```diff
@@ -40,28 +40,27 @@ RUN set -ex; \
 	cd build; \
 	emcmake cmake \
 		-DCMAKE_BUILD_TYPE=MinSizeRel \
-		-DCMAKE_INSTALL_PREFIX=/emsdk/emscripten/sdk/system/ \
+		-DCMAKE_INSTALL_PREFIX=/emsdk/upstream/emscripten/system \
 		-DZ3_BUILD_LIBZ3_SHARED=OFF \
 		-DZ3_ENABLE_EXAMPLE_TARGETS=OFF \
 		-DZ3_BUILD_TEST_EXECUTABLES=OFF \
 		-DZ3_BUILD_EXECUTABLE=OFF \
 		-DZ3_SINGLE_THREADED=ON \
 		-DCMAKE_CXX_FLAGS="-s DISABLE_EXCEPTION_CATCHING=0" \
 		..; \
-	make; make install; \
+	make ; make install; \
 	rm -r /usr/src/z3; \
 	cd /usr/src; \
-	wget -q 'https://dl.bintray.com/boostorg/release/1.73.0/source/boost_1_73_0.tar.bz2' -O boost.tar.bz2; \
-	test "$(sha256sum boost.tar.bz2)" = "4eb3b8d442b426dc35346235c8733b5ae35ba431690e38c6a8263dce9fcbb402  boost.tar.bz2"; \
+	wget -q 'https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.bz2' -O boost.tar.bz2; \
```

Comment:
Why is the boost upgrade needed?

Reply:
Because there's a new boost version ;-).

Reply:
We've done it at the same time in the past as well - there's little reason to expect issues with it and it halves the amount of work updating all this is.